dev-tools/composer.json

```diff
@@ -1,12 +1,11 @@
 {
     "require": {
-        "php": "^7.1"
+        "php": "^7.4 || ^8.0"
```

Comment:
maybe just ^8.0?

Reply:
@keradus 
yes - with the composer.json split you can see now
no - without extracting humbug as deployment job is don on PHP 7.4

the split is necessary for now as humug/box and composer-require-checker are incompatible versions of their deps (humbug/php-scoper, dependency of humbug/box needs symfony/console ` ^3.2 || ^4.0` while maglnet/composer-require-checker needs `^5.2.6`)